node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageReceiver.scala

```diff
@@ -202,9 +202,14 @@ class PeerMessageReceiver(
         //we don't want to have to request them manually
         sender.sendHeadersMessage()
         Future.successful(this)
-      case _: AddrMessage =>
+      case addr: AddrMessage =>
+        addr.addresses.foreach(node.createInDbIfBlockFilterPeer)
```

Comment:
Yeah, most likely the node has changed config since the last time that peer saw them. I think in that case we should just disconnect from them and find another one